src/Simplex/Chat/Protocol.hs

```diff
@@ -1201,8 +1222,16 @@ data ContactShortLinkData = ContactShortLinkData
   }
   deriving (Show)
 
+-- TODO [chat relays] owner's public key for signing group actions and messages
 data GroupShortLinkData = GroupShortLinkData
-  { groupProfile :: GroupProfile
+  { groupProfile :: GroupProfile,
+    chatRelays :: [ChatRelayInfo]
```

Comment:
this is all aready typed in agent, don't reinvent it - there is a more general pattern of address delegation.